loop-fs-iso-impl/src/main/java/com/github/stephenc/javaisotools/loopfs/iso9660/EntryInputStream.java

```diff
@@ -107,13 +107,13 @@ public long skip(final long n) {
     }
 
     public int available() {
-        return Math.max(this.rem, 0);
+        return (int) Math.max(this.rem, 0);
```

Comment:
When `this.rem` is greater than Integer.MAX_VALUE, this will return a negative number. I would write it more like this:

```
if (this.rem > Integer.MAX_VALUE) {
    return Integer.MAX_VALUE;
} else if (this.rem < 0) {
    return 0;
} else {
    return (int) this.rem;
}
```